### ISSUE TITLE
Broken on clean install

### ISSUE THREAD

**Issue by AndrewSB** [soheilbm/Firebase#28]
Tracking on Firebase over at https://github.com/firebase/firebase-ios-sdk/issues/600

A clean install (https://github.com/AndrewSB/broken-carthage-maybe) doesn't work. 

@soheilbm am I doing something obviously stupid? I tried to follow the README as closely as possible

**Comment by AndrewSB** [comment]
As an update, it definitely seems to be something broken with this repo or with my integration steps.

A manual drag and drop installation from Firebase's docs compiles and builds (attempted over at https://github.com/AndrewSB/broken-carthage-maybe/tree/manual-integration)

**Comment by brooksdubois** [comment]
Agreed, I followed all the steps very closely and both the Firebase Database frameworks are not recognized. Very disappointing. The analytics framework seems to work okay.

**Comment by paynerc** [comment]
@AndrewSB Are you running on High Sierra by any chance? Up until recently this was working fine for me. And then this week I upgraded to High Sierra and now I am seeing this issue. Manually dropping in the 4.10.0 frameworks from Firebase works, but that is not something that is going to work in our build pipeline. One of the things I read about this issue is that there is a version mismatch between two libraries and that is causing the issue. But I don't know why it just recently surfaced, unless the Xcode Command Line tools for High Sierra are just different enough...

**Comment by paynerc** [comment]
@AndrewSB I opened a PR on this repo to have it consume the 4.10.0 frameworks. I am now able to link past that error I was seeing before. Now my app crashes on launch, but its compiling. I was wondering if you are seeing similar after pulling with the latest?

**Comment by AndrewSB** [comment]
Yup, I was on high sierra. Firebase has official (beta) support for Carthage now though, so you should just be able to use their setup. See https://github.com/firebase/firebase-ios-sdk/issues/9#issuecomment-367871385

**Comment by schlingding** [comment]
I’m still supporting https://github.com/MonsantoCo/FirebaseFramework which is up to date with 4.10.0 and the latest Crashlytics until there is a better solution.

The current Firebase support for Carthage is still not enough.
